x-pack/plugins/osquery/public/routes/saved_queries/edit/tabs.tsx

```diff
@@ -61,18 +72,29 @@ const ResultTabsComponent: React.FC<ResultTabsProps> = ({
         ) : null,
       },
     ],
-    [actionId, agentIds, ecsMapping, startDate, endDate, addToTimeline, failedAgentsCount]
+    [
+      actionId,
+      agentIds,
+      ecsMapping,
+      startDate,
+      endDate,
+      addToTimeline,
+      addToCase,
+      failedAgentsCount,
+    ]
   );
 
   return (
-    <EuiTabbedContent
-      // TODO: extend the EuiTabbedContent component to support EuiTabs props
-      // bottomBorder={false}
-      tabs={tabs}
-      initialSelectedTab={tabs[0]}
-      autoFocus="selected"
-      expand={false}
-    />
+    <CasesContext owner={CASES_OWNER} permissions={casePermissions}>
+      <EuiTabbedContent
+        // TODO: extend the EuiTabbedContent component to support EuiTabs props
+        // bottomBorder={false}
```

Comment:
nit: remove commented out code

Reply:
I'd leave the TODO, maybe there's gonna be time to take care of it soon